### ISSUE TITLE
Add vehicle AC which combines heater and cooler, and is found in most modern vehicles.

### ISSUE THREAD

**Issue by meelock** [CleverRaven/Cataclysm-DDA#46375]
# Is your feature request related to a problem? Please describe.
Many new survivors and the like have difficulty with hot and cold weather for various reasons, and vehicle AC units are in most cars (even ancient ones) in real life.

# Describe the solution you'd like
Add a vehicle AC that consumes a constant (or variable if that is more realistic) amount of electricity from the vehicle to make the environment COMFORTABLE. These AC units would be damaged or destroyed most of the time, when found in cars, as their filters and other systems could not withstand the cataclysm.

# Describe alternatives you've considered
Allow heater and cooler units to be installed in the same space, or add a small version of the heater and cooler units that would be installed in most cars (though often damaged). Nesimtitul suggested naming it "vehicle climate control."

**Comment by Mom-Bun** [comment]
Cars are already air conditioned AFAIK, but I'd love if this change was accompanied with having temperature in cars not being constant, so you are encouraged to use the AC

**Comment by anothersimulacrum** [comment]
There is, but it's very much non-obvious and kind of hidden.
Here's 30 minutes waited in a house:
![image](https://user-images.githubusercontent.com/42699974/103246221-92266f80-4917-11eb-939a-542bc4624c5b.png)
30 minutes outside:
![image](https://user-images.githubusercontent.com/42699974/103246236-9bafd780-4917-11eb-8e41-33e880c54cc9.png)
30 minutes in a car (that's turned on, I had it work in a car that was off earlier but it did not work now):
![image](https://user-images.githubusercontent.com/42699974/103246291-d285ed80-4917-11eb-9089-b7ae6652e432.png)

Here's a save where I tested it:
[Town.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/5748966/Town.zip)

tl;dr: This mechanic already exists, but it could definitely use clarifying.

**Comment by PaulBlay** [comment]
> There is, but it's very much non-obvious and kind of hidden.

Maybe should be included in the ^ vehicle electronics you can turn on and off.  How well it works, and whether it can be used without an engine running, would depend on the car model in question, ideally.

**Comment by NeviNovat** [comment]
Or make it so that it uses battery ,free permanent AC kinda makes winter/summer easy from a clothing point of view. And can save characters from freezing to death like in my case.

**Comment by emptytriangle** [comment]
Note the vehicle cooler is NOT an a/c unit and will freeze the interior of the vehicle to sub-zero

**Comment by PaulBlay** [comment]
@emptytriangle That kinda sounds like a bug to me. If a cooler is doing something more sophisticated than 'fan air in from outside' then it would probably have a thermostat (temperature sensor + setting) attached rather than just cooling as much as it can.

**Comment by emptytriangle** [comment]
>If a cooler is doing something more sophisticated than 'fan air in from outside' then it would probably have a thermostat (temperature sensor + setting) attached rather than just cooling as much as it can.

The cooler is definitely doing more than "fan air in from outside". It has a compressor and phase-change coolant loop to pump heat from inside the vehicle to outside.

> That kinda sounds like a bug to me. 
I only started using the vehicle cooler near the 0.E wrapup and later, so I assumed it was originally meant to be a box-truck refrigeration unit. People used to use these to make walk-in freezers in their bases. So I looked at #30720 which seems to have implemented them, and it is definitely supposed to be a cooler to cool humans.

So yes, vehicle cooler cooling to subzero is not intentional. And the A/C unit should probably have a thermostat so you can set the target temp (something I wanted to have for my old walk-in freezer, it used to chill to lower than -20C).

**Comment by emptytriangle** [comment]
looks like cooler was changed from emitting fd_cold_air2 to fd_cold_air3 in #31888

I suspect this was when vehicle coolers began to freeze things

**Comment by Zigoos** [comment]
If CDDA is in the near future, then we can assume that cars use slightly more advanced peltier elements. This will allow you to create a system only from a thermometer and a smart controller?

**Comment by Night-Pryanik** [comment]
Closing as stale, since stalebot can't do this by itself.
